Dockerfile

```diff
@@ -1,19 +1,18 @@
-FROM rust:1.60.0
+FROM rust:1.60-slim-bullseye
```

Comment:
would rec testing building this + running on one of our rpcs, might impact dynamic libraries